ark/phenotyping/som_utils.py

```diff
@@ -391,13 +445,19 @@ def create_fov_pixel_data(fov, channels, img_data, seg_labels,
     seg_labels_flat = seg_labels.flatten()
     pixel_mat['segmentation_label'] = seg_labels_flat
 
+    # remove any rows with channels that sum to zero prior to sampling
+    pixel_mat = pixel_mat.loc[(pixel_mat[channels] != 0).any(1), :]
+
+    # normalize the row sums of pixel mat
+    pixel_mat = normalize_rows(pixel_mat, channels)
+
     # subset the pixel matrix for training
     pixel_mat_subset = pixel_mat.sample(frac=subset_proportion, random_state=seed)
 
     return pixel_mat, pixel_mat_subset
 
 
-def create_pixel_matrix(fovs, channels, base_dir, tiff_dir, seg_dir,
+def create_pixel_matrix(fovs, channels, base_dir, tiff_dir, seg_dir, seg_suffix='_feature_0.tif',
```

Comment:
It would be be helpful to have an "img_sub_folder" argument here, to be fed into "load_utils.load_imgs_from_tree." Had to hard code it in when I was testing which was kind of annoying